src/components/Menu/Menu.tsx

```diff
@@ -75,8 +75,9 @@ export const Menu = ({
           className={menuClassNames}
           defaultOpenKeys={defaultOpenKeys}
           defaultSelectedKeys={defaultSelectedKey ? [defaultSelectedKey] : undefined}
+          getPopupContainer={() => document.querySelector(`.${menu}`)!}
```

Comment:
what does this prop do?

Reply:
it is necessary to allow you to inherit the css. If the popover is hung in the bodysuit, we can't stylize it